src/views/Callback.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+  <div class="callback">Callback</div>
+</template>
+
+<script>
+export default {
+  name: 'callback',
+  mounted() {
+    this.$auth.handleAuthentication().then((data) => {
+      this.$router.push({ name: 'home' })
```

Comment:
Despite the handleAuthentication() call here, I don't think there's any real test that the user successfully authenticates. The router.push() happens regardless 😕

Reply:
Resolved with router-view tag in App.vue

Reply:
🤔